variables.tf

```diff
@@ -1,3 +1,27 @@
+variable "defaults" {
+  type        = any
+  description = "Overwrite defaults for various repository settings"
+  default     = {}
+
+  # Example:
+  # defaults =
```

Comment:
opening bracket is missing

# defaults = {

Reply:
Good catch

Reply:
fixed